src/js/components/Drop/stories/TrapFocus.js

```diff
@@ -23,13 +23,14 @@ const TooltipButton = ({ icon, name }) => {
       </Button>
       {ref.current && over && (
```

Comment:
Any reason we aren't using `<Tip />`?

Reply:
Good catch, I think I've lost it while merging conflicts on this story. I'll roll it back

Reply:
I added a comment. Let me know what you think.